concent_api/core/views.py

```diff
@@ -532,7 +536,6 @@ def get_file_status(file_transfer_token_from_database: message.concents.FileTran
 
     current_time = int(datetime.datetime.now().timestamp())
     file_transfer_token = message.concents.FileTransferToken(
-        timestamp                       = current_time,
         token_expiration_deadline       = current_time + settings.TOKEN_EXPIRATION_DEADLINE,
```

Comment:
Please add an `assert` to check that timestamp <= deadline.

Reply:
:+1: